app/src/main/java/com/pileproject/drive/util/bluetooth/RxBluetoothConnector.java

```diff
@@ -77,11 +81,47 @@ public void call(Subscriber<? super BluetoothSocket> subscriber) {
 
                     subscriber.onNext(socket);
                     subscriber.onCompleted();
-                } catch (IOException e) {
-                    subscriber.onError(e);
+                } catch (IOException firstCause) {
+
+                    try {
+                        BluetoothSocket socket = connectByRfcommSocket(bluetoothDevice);
+                        subscriber.onNext(socket);
```

Comment:
Oh, I missed the point. So, what is the difficulty to use this method in `BluetoothCommunicator#open`? I think a note for that will help us to consider whether this method is necessary or not in the future :)